src/test/java/org/entando/plugins/pda/core/model/FakeTask.java

```diff
@@ -0,0 +1,52 @@
+package org.entando.plugins.pda.core.model;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Singular;
+
+public class FakeTask extends Task {
```

Comment:
Rename to MockTask (and the builder too)

Reply:
In my opinion they should be called `Fake` due to the design pattern difference compared with `Mock`:
https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing

In this case, the `FakeEngine` is a fully working engine, however with simplified functionalities.